src/Tasks/Microsoft.Common.props

```diff
@@ -164,4 +164,14 @@ Copyright (C) Microsoft Corporation. All rights reserved.
     <Import Project="$(MSBuildUserExtensionsPath)\$(MSBuildToolsVersion)\Imports\Microsoft.Common.props\ImportAfter\*" Condition="'$(ImportUserLocationsByWildcardAfterMicrosoftCommonProps)' == 'true' and exists('$(MSBuildUserExtensionsPath)\$(MSBuildToolsVersion)\Imports\Microsoft.Common.props\ImportAfter')"/>
     <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Imports\Microsoft.Common.props\ImportAfter\*" Condition="'$(ImportByWildcardAfterMicrosoftCommonProps)' == 'true' and exists('$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Imports\Microsoft.Common.props\ImportAfter')"/>
   </ImportGroup>
+
+  <!-- 
+        Import NuGet.props file.
+    -->
+  <PropertyGroup>
+    <IsRunningFromVisualStudio Condition="'$(IsRunningFromVisualStudio)'==''">$([MSBuild]::IsRunningFromVisualStudio())</IsRunningFromVisualStudio>
+    <NuGetProps Condition="'$(NuGetProps)'=='' and '$(IsRunningFromVisualStudio)'=='true'">$(MSBuildToolsPath32)\..\..\..\Common7\IDE\CommonExtensions\Microsoft\NuGet\NuGet.props</NuGetProps>
+    <NuGetProps Condition="'$(NuGetProps)'==''">$(MSBuildToolsPath)\NuGet.props</NuGetProps>
+  </PropertyGroup>
+  <Import Condition="Exists('$(NuGetProps)')" Project="$(NuGetProps)" />
```

Comment:
_Should_ this be conditioned? We now expect it to be part of every MSBuild distribution, right? And NuGet is licensed compatibly so that people could distribute it?

Reply:
If the question is if we should have 
``` xml
Condition="Exists('$(NuGetProps)')"
```

I think that is good to not try importing without checking the existence of the file.

Reply:
For files we expect to preserve core functionality, I don't think that's true: it transforms a semi-actionable error "NuGet.props doesn't exist" into hidden behavior changes like "my project-wide package versions aren't being respected".

Reply:
I see your point. Removed.

Reply:
May have been a good idea in theory but I just undid it since it caused our bootstrap builds to fail. This also means there's no strong order dependency between merging this and distributing NuGet.props in your packages.